internal/provider/utils.go

```diff
@@ -983,3 +983,54 @@ func canUpdateEntityNameBusinessMetadata(entityType, oldEntityName, newEntityNam
 		return false
 	}
 }
+
+const (
+	maxFileSize     = 1024 * 1024 * 1024 // 1GB
+	maxFileSizeInGB = 1
+)
+
+func UploadFileToAzureBlob(url, filePath, contentFormat string) error {
```

Comment:
Could you add a comment here to indicate it was copied from CLI repo and reference a JIRA to export this method instead? Thank you!